_includes/head.html

```diff
@@ -23,4 +23,5 @@
   {% endif %}
   <link rel="stylesheet" href="/static/css/additional-styles.css">
   <script src="/static/js/header-links.js" defer></script>
+  <link rel="alternate" type="application/atom+xml" title="{{ site.title }} News" href="/atom.xml">
```

Comment:
For a nitpick I would add this after the `{%- seo -%}` tag since they feel related